cfgmgr/stpmgr.cpp

```diff
@@ -0,0 +1,1109 @@
+/* copyright 2019 Broadcom. The term "Broadcom" refers to Broadcom Inc. and/or
```

Comment:
This copyright statement is not in the stporch.cpp/h
Can we keep it consistent across diff files.

Reply:
Done